lexicons/app/bsky/unspecced/getSuggestionsSkeleton.json

```diff
@@ -19,7 +19,12 @@
             "maximum": 100,
             "default": 50
           },
-          "cursor": { "type": "string" }
+          "cursor": { "type": "string" },
+          "relative_to_did": {
```

Comment:
I think this is the only place `snake_case` is used as a lexicon field—could we bring this in line with the `camelCase` convention?

Reply:
Oh sure